pkg/workflow/workload/pod/builder.go

```diff
@@ -236,16 +236,6 @@ func (m *Builder) CreateVolumes() error {
 		},
 	})
 
-	// Add host volume for host docker socket file for coordinator container.
-	m.pod.Spec.Volumes = append(m.pod.Spec.Volumes, corev1.Volume{
-		Name: common.HostDockerSockVolumeName,
```

Comment:
Can we delete `common.HostDockerSockVolumeName` or docker related const now?

Reply:
removed